dart/lib/src/sentry_options.dart

```diff
@@ -136,6 +136,15 @@ class SentryOptions {
   /// example : ['sentry'] will include exception from 'package:sentry/sentry.dart'
   List<String> get inAppIncludes => List.unmodifiable(_inAppIncludes);
 
+  /// Configures wether stack trace frames are considered in app frames as default.
+  /// You can use this to essentially make [inAppIncludes] or [inAppExcludes]
+  /// a white or black list.
```

Comment:
allow/deny list instead https://develop.sentry.dev/inclusion/

Reply:
Dang, I forgot about that :(